CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
-## v0.1.1 (24 Nov 2022)
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
```

Comment:
I'm good with following a guideline.
I guess current format was from the vscode python extension
https://github.com/microsoft/vscode-python/blob/main/CHANGELOG.md
Though we're not strictly follow it and there's no need to follow it.
I'm good with the "Keep a Changlog". But looks like this pr doesn't modify existing date format and types of changes. Do we want to change them all? Do we want to link the format before we fully follow it?

Reply:
Should we create an issue for this item to form discussion around?

Reply:
I missed the dates. Perhaps we can:
 - Change the headers (versions & dates)
 - Get rid of the link to keep a changelog
 - create an issue to discuss the standard and whether we want to apply that to the version notes going forward, or retroactively apply it to things that've already been released? 

I also noticed I've not updated the Changelog as part of this PR 🤦

Reply:
sounds good to me